### ISSUE TITLE
Ban command uses int instead of a long which prevents you from banning by steamid

### ISSUE THREAD

**Issue by MatStef132** [Facepunch/sbox-issues#2235]
### Describe the bug

When I try to ban someone from the console it always states that it can't find that user. When you try to enter the correct format (SteamID64) it converts it to a smaller number. Someone on the discord told me that's it's because the id is an int instead of a long.

### To Reproduce

1. Go on a server with someone
2. Open the console
3. Try banning them by typing banid 0 steamid64

### Expected behavior

The player shout be banned

### Media/Files

![unknown](https://user-images.githubusercontent.com/50845552/187046056-b41c49d2-e0ef-4dad-9fd2-b9e6484afce5.png)
![unknown](https://user-images.githubusercontent.com/50845552/187046063-1cee1da7-e3d1-4edb-b189-f69e6f5cbc49.png)

### Additional context

I'm running windows server 2022, my pc specs are: ryzen 5 3600, 64 GB DDR4 ram.

**Comment by DevulTj** [comment]
I don't think we should be providing administrative features outside of games.

**Comment by MatStef132** [comment]
> I don't think we should be providing administrative features outside of games.

Banning of players is a pretty universal feature and it would be nice if it was implemented, even basic, in the game 👍
The command is already there it probably is implemented and just needs a fix.

**Comment by DevulTj** [comment]
It's a collection of code that is over 15 years old, I think we're better off removing it - which is now the case in the main game branch.

I've added `Game.ShouldConnect` which will allow you to create your own banning system.
